backend/src/main/resources/db/testdata/V666.15__insert_dummy_vigilance_area.sql

```diff
@@ -1,60 +1,196 @@
+DO
+$$
+    DECLARE
+        today                                        DATE;
+        date_within_year_and_3_months                DATE;
+        date_within_quarter                          DATE;
+        date_within_year_not_in_quarter_nor_3_months DATE;
+    BEGIN
+        -- Définir la date d'aujourd'hui
+        today := date_trunc('day', CURRENT_DATE);
 
-INSERT INTO public.vigilance_areas(id, comments, created_by, end_date_period, ending_condition, ending_occurrence_date, ending_occurrence_number, frequency, geom, is_draft, links, name,source,
-start_date_period, themes, visibility,linked_amps, linked_regulatory_areas, computed_end_date)
-VALUES (2, 'Des dauphins partout', 'DEF', '2024-03-01 23:59:59.99999', 'END_DATE', '2030-03-01 23:59:59.99999', NULL, 'ALL_WEEKS', '0106000020E610000001000000010300000001000000050000009E64CAC84F5DFDBF447087DF4CDA4840FAA68553DF2CFDBFE8B528CA7AD84840B2FDB77AA327FCBF30AA4C29CDD64840D9ADABB07A86FBBFE4090ED908D748409E64CAC84F5DFDBF447087DF4CDA4840', false, NULL, 'Zone de vigilance 2', 'Unité PAM Themis', '2024-02-27 00:00:00.00000', '{"AMP","Mixte"}', 'PRIVATE', '{}', '{"625", "425"}', '2030-03-01 23:59:59.99999');
+-- 1. Date dans l'année courante, mais pas dans le trimestre actuel ni dans les trois premiers mois de l'année
+        date_within_year_not_in_quarter_nor_3_months :=
+            (CASE
+                 WHEN EXTRACT(MONTH FROM CURRENT_DATE) <= 3
+                     THEN date_trunc('year', CURRENT_DATE) + INTERVAL '9 month' -- Pour éviter Q1 et les 3 premiers mois
+                 WHEN EXTRACT(MONTH FROM CURRENT_DATE) <= 6
+                     THEN date_trunc('year', CURRENT_DATE) + INTERVAL '6 month' -- Pour éviter Q2 et les 3 premiers mois
+                 WHEN EXTRACT(MONTH FROM CURRENT_DATE) <= 9
+                     THEN date_trunc('year', CURRENT_DATE) + INTERVAL '3 month' -- Pour éviter Q3 et les 3 premiers mois
+                 ELSE date_trunc('year', CURRENT_DATE) -- Pour éviter Q4
```

Comment:
Dans ce cas (mois courant entre octobre et décembre) la date `date_within_year_not_in_quarter_nor_3_months` sera le 1er janvier de l'année en cours, ce qui ne correspond pas à une "Date dans l'année courante, mais pas dans le trimestre actuel ni dans les trois premiers mois de l'année" a priori ?

Reply:
C'est pas les trois premiers mois de l'année mais les trois prochains mois

Reply:
J'ai noté juste un probleme quand on sera le 31 decembre / 1 er janvier mais on ne devrait rien merger à ce moment là de l'année normalement ^^

Reply:
Okay !
Peut-être plus simplement on pourrait ajouter 6 mois et boucler quand ça dépasse 12 avec un modulo `%` :
```SQL
date_within_year_not_in_quarter_nor_3_months = SELECT make_date(
    EXTRACT(year FROM CURRENT_DATE)::INTEGER,
    (EXTRACT(month FROM CURRENT_DATE) + 6 - 1)::INTEGER % 12 + 1,
    1
)
```